pegasus/forms/hoc_signup_2017.rb

```diff
@@ -1,6 +1,33 @@
 require pegasus_dir 'forms/hoc_signup_2014'
 
 class HocSignup2017 < HocSignup2014
+  def self.normalize(data)
+    super
+    result = super
```

Comment:
you should be able to remove the first call to `super`, since you're capturing the return value on the second call.